osv/malicious/npm/bugsnagmw/MAL-0000-bugsnagmw.json

```diff
@@ -0,0 +1,45 @@
+{
+  "modified": "2024-03-31T14:28:23Z",
+  "published": "2024-03-31T14:28:23Z",
+  "schema_version": "1.5.0",
+  "id": "",
+  "summary": "Malicious code in bugsnagmw (npm)",
+  "details": "The package `bugsnagmw` (npm) version 1.0.3 contains malicious code. The code was obfuscated to avoid detection. The malicious code is designed to steal sensitive information from the user's environment and send it to a remote server. See https://stacklok.com/blog/analysis-of-an-obfuscated-code-attack-on-npms-package-registry for more details.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "npm",
+        "name": "bugsnagmw"
+      },
+      "versions": [
+        "1.0.0",
+        "1.0.1",
+        "1.0.2",
+        "1.0.3"
+      ]
+    }
+  ],
+  "credits": [
+    {
+      "name": "Stacklok: trustypkg.dev",
+      "type": "FINDER",
+      "contact": [
+        "https://discord.com/invite/RkzVuTp3WK"
+      ]
+    }
+  ],
+  "database_specific": {
+    "malicious-packages-origins": [
+      {
+        "sha256": "",
+        "source": "trustypkg.dev",
+        "versions": [
+          "1.0.0",
+          "1.0.1",
+          "1.0.2",
+          "1.0.3"
+        ]
+      }
+    ]
+  }
```

Comment:
This section isn't required if this is a hand-crafted OSV file.

It's largely there to manage ingestion from a bulk collection of upstream OSV files to detect whether or not the OSV has been ingested before.

Without the sha256 this will do weird things to the details section.

Finally if you do wish to do bulk or automated ingestion I am happy to agree on a source ID that works for you.